Zend/zend_builtin_functions.c

```diff
@@ -1656,6 +1660,16 @@ ZEND_FUNCTION(set_error_handler)
 }
 /* }}} */
 
+/* {{{ proto string get_error_handler(void)
+   Returns the currently defined error handler, or null */
+ZEND_FUNCTION(get_error_handler)
+{
```

Comment:
hehe, nice catch, was copying most of the code from restore_error_handler() and surprisingly zend_parse_parameters_none() is not used there.